MAINTENANCE.md

```diff
@@ -4,11 +4,11 @@
 cd portal/
 git submodule update --init # if necessary
 cd submodules/motoko
-git checkout 0.7.6 #or whatever new tag you desire
+git checkout 0.14.1 #or whatever new tag you desire
 cd ../..
 git add submodules/motoko # add the change to the submodule
 ```
-Now replace `static/moc-interpreter-0.13.6.js` with `static/moc_interpreter-0.13.7.js`, downloaded from Motoko release page. It would be better if this wasn't checked in, but, for now, it is.
+Now replace static/moc-interpreter-0.14.0.js with static/moc_interpreter-0.14.1.js, downloaded from motoko release page. It would be better if this wasn't checked in, but, for now, it is.
```

Comment:
```suggestion
Now replace static/moc-interpreter-0.14.0.js with static/moc_interpreter-0.14.1.js, downloaded from Motoko release page. It would be better if this wasn't checked in, but, for now, it is.
```